test/unit/feedbackView_spec.js

```diff
@@ -58,15 +58,14 @@ describe('FeedbackView', () => {
       videoScore.selectedIndex = reportLevel;
       sendButton.click();
 
-      expect(Modal.hide.calledOnce).to.be.true;
-
-      expect(Utils.sendEvent.calledOnce).to.be.true;
+      expect(Utils.sendEvent.called).to.be.true;
       expect(Utils.sendEvent.getCall(0).args[0]).to.be.equal('feedbackView:sendFeedback');
       expect(Utils.sendEvent.getCall(0).args[1]).to.be.deep.equal({
         audioScore: audioScore.options[audioScore.selectedIndex].value,
         videoScore: videoScore.options[videoScore.selectedIndex].value,
         description: otherInfo.value,
       });
+      expect(Utils.sendEvent.getCall(1).args[0]).to.be.equal('feedbackView:reportIssue');
```

Comment:
Shouldn't this _not_ report and issue because the level isn't low enough? (comments in line 55, 56)